rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -275,6 +277,27 @@ public Repository getRepository() {
         return repository;
     }
 
+    public BlockExecutorFactory getBlockExecutorFactory() {
```

Comment:
Could we perform a little cleanup in the `RskTestFactory`? I mean something like pulling up `getTransactionExecutorFactory` and using it to create the `BlockExecutor` factory

Reply:
The creation of BlockExecutor in BlockExecutorFactory should use a ProgramTraceProcessor, that is provided at the last moment, and cannot use the logic of RskTestFactory.getTransactionExecutorFactory